src/main/java/frc/robot/subsystems/vision/Vision.java

```diff
@@ -0,0 +1,75 @@
+package frc.robot.subsystems.vision;
+
+import org.littletonrobotics.junction.Logger;
+
+import edu.wpi.first.math.Matrix;
+import edu.wpi.first.math.numbers.N1;
+import edu.wpi.first.math.numbers.N3;
+import edu.wpi.first.math.VecBuilder;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.subsystems.drive.Drive;
+import edu.wpi.first.math.util.Units;
+import java.util.Arrays;
+
+public class Vision extends SubsystemBase
+{
+    private final VisionIO                 _io;
+    private final VisionIOInputsAutoLogged _inputs           = new VisionIOInputsAutoLogged();
+    private final Drive                    _drive;
+    private final PIDController            _rotatePID;
+    private double                         _maxSpeed         = 0.3;
+    private double                         _lastTimestamp    = 0.0;
+    private boolean                        _targetingEnabled = false;
```

Comment:
These look like they're carryovers from some code we had last year. I'm not sure if we're going to need it this year since with the better camera we might be able to rely entirely on odometry, and I'd like to not include code we're not using in the codebase.

Let's remove these variables and all of their associated parts of the rest of the subsystem. If we need them later, we can add them back in then.